### DISCUSSION TITLE
Issue with material UI date

### DISCUSSION THREAD

**Discussion by sivaputrevu** [react-hook-form/react-hook-form#2353] [Q&A]
Hi I am uaing material UI date picker[KeyBoard Date Picker] and I want to validate if the submit button clicked to highlight the date field if the date is empty. If possible help me with the below items. a form having 4 mandatory fields, untill the mandatory fields are filled the submit shouldn't be enabled! Material UI pickers version 3.2.10 and material ui core 4.11.0

**Comment by bluebill1049** [comment]
needs more detail, do you have a codesandbox?
